src/object.d

```diff
@@ -1737,7 +1728,7 @@ class Throwable : Object
                     sink("\n"); sink(t);
                 }
             }
-            catch (Throwable)
+            catch (Exception)
             {
```

Comment:
Is it even at any time a good idea to silently catch Exceptions?  It seems to me a bit suspicious that `toString` doesn't expose the `nothrow` guarantee publicly, but in the other hand does this silent catching..